src/components/ApplicationForm/ReviewCards.js

```diff
@@ -0,0 +1,297 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import holo from '../../assets/holo_review.svg'
+import Banner from '../Banner'
+import { H1, P, QuestionHeading, A } from '../Typography'
+import { Button, Checkbox } from '../Input'
+
+const HoloBackground = styled.img`
+  position: absolute;
+  text-align: center;
+  display: block;
+  float: left;
+  overflow-x: hidden;
+  margin: 0 auto;
+  opacity: 0.8;
+  width: 100%;
+  z-index: -1;
+  top: 0;
+  left: 0;
+`
+
+const ReviewContainer = styled.div`
+  position: relative;
+  max-width: 800px;
+  margin: 3em auto;
+`
+
+const ContentWrapper = styled.div`
+  ${p =>
+    p.grid &&
+    `display: grid;
+grid-template-columns: auto auto;
+grid-template-rows: auto auto auto auto auto auto;
+grid-gap: 0;`}
+  padding: ${p => (p.textBlock ? `0.5em 0` : `2em`)};
+`
+
+const InfoGroupWrapper = styled.div`
+  padding: 0.5em;
+`
+
+const StyledH1 = styled(H1)`
+  color: #18cdcd;
+`
+
+const StyledBanner = styled(Banner)`
+  && {
+    max-width: 800px;
+    top: 18em;
+    padding: 0;
+    z-index: 0;
+  }
+`
+
+const JohnDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const CenterH1 = styled(H1)`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`
+
+const RequiredAsterisk = styled.span`
+  color: ${p => p.theme.colors.warning};
+`
+
+const InfoGroup = ({ heading, data }) => (
+  <InfoGroupWrapper>
+    <H1 size="1.2em">{heading}</H1>
+    <StyledH1 size="1.5em">{data}</StyledH1>
+  </InfoGroupWrapper>
+)
+
+const ethnicityOptions = {
+  asian: 'Asian',
+  black: 'Black or African American',
+  caucasian: 'Caucasian or European',
+  hispanic: 'Hispanic or Latinx',
+  middleEastern: 'Middle Eastern',
+  nativeHawaiian: 'Native Hawaiian or Pacific Islander',
+  northAmerica: 'North American Indigenous',
+  other: 'Other',
+  preferNot: 'Prefer not to say',
+}
+
+const getEthnicities = obj => Object.keys(obj).filter(key => obj[key])
+
+export default ({ formInputs, handleEdit }) => {
+  const [termsAndConditions, setTermsAndConditions] = useState(formInputs.termsAndConditions)
+
+  const ethnicities = getEthnicities(formInputs.basicInfo.ethnicity).map(e => ethnicityOptions[e])
+  var ethnicitiesVal = ''
+
+  for (var i = 0; i < ethnicities.length; i++) {
+    ethnicitiesVal = ethnicitiesVal.concat(ethnicities[i])
+
+    if (i < ethnicities.length - 1) {
+      ethnicitiesVal = ethnicitiesVal.concat(', ')
+    }
+  }
+
+  var attendedVal = ''
+  for (var j = 0; j < formInputs.questionnaire.eventsAttended.length; j++) {
+    attendedVal = attendedVal.concat(formInputs.questionnaire.eventsAttended[j])
+
+    if (i < formInputs.questionnaire.eventsAttended.length - 1) {
+      attendedVal = attendedVal.concat(', ')
+    }
+  }
```

Comment:
hmm it might be better to just store these as an array.

Reply:
agree with what you said! but just a heads up though @vincentchiang1 the options and values for eventsAttended might be mapped differently so you might have to adjust based on @kevin-zou's implementation in part 3 https://github.com/nwplus/livesite/pull/157/files (I believe he's using option numbers so you night need use his map from part 3)
```
const options = [
    { value: '1', label: 'MLH' },
    { value: '2', label: 'Social media' },
    { value: '3', label: 'Website' },
    { value: '4', label: 'Word of mouth' },
    { value: '5', label: 'Club newsletter' },
    { value: '6', label: 'Faculty newsletter' },
    { value: '7', label: 'Professor/in class' },
    { value: '8', label: 'Other' },
  ]
```
since this PR is addressing the review UI only, what you have is good. when displaying the actual data from the HackerApplicationContext in your next PR, you'll probably need to make use of the map

Reply:
sounds good thanks yall